Automattic/wp-calypso
# Update Plan ThankYou/Congrats page

Fixes https://github.com/Automattic/dotcom-forge/issues/2900

## Proposed Changes

* Update plans thank you page to redesign V2

| Before  | After |
| :-------------: | :-------------: |
| <img src="https://github.com/Automattic/wp-calypso/assets/6586048/6e1745e0-9f61-4bfc-9fba-36e0e0146272">  | <img src="https://github.com/Automattic/wp-calypso/assets/6586048/035309e9-e89d-43d9-990f-b41cd64422d5">|

Redesign v2 Mobile
![mobile](https://github.com/Automattic/wp-calypso/assets/6586048/94a5200d-478c-4957-b7ef-17ea88a5f551)


## Testing Instructions



* Select a WordPress.com plan and purchase it
* Upsells should still work, you can click on "No thanks" and it should show this ThankYou page
* Check the ThankYou page for mobile/desktop
* Test around different purchases (email/Google/etc.) try to break it
* See if tracks events are triggered

## Pre-merge Checklist
